src/query/anonymization.c

```diff
@@ -565,12 +565,40 @@ static hash_t hash_label(Oid type, Datum value, bool is_null)
   return hash;
 }
 
+static ParamExternData *get_param_data(ParamListInfo bound_params, int paramid)
+{
+  return bound_params->paramFetch != NULL ? bound_params->paramFetch(bound_params, paramid, true, NULL) : &bound_params->params[paramid];
+}
+
+static hash_t label_hash(Node *node, ParamListInfo bound_params)
+{
+  if (IsA(node, Const))
+  {
+    Const *label = castNode(Const, node);
+    return hash_label(label->consttype, label->constvalue, label->constisnull);
+  }
+  else if (IsA(node, Param))
+  {
+    Param *label = castNode(Param, node);
+#if PG_MAJORVERSION_NUM == 13
+    ParamExternData *param_data = get_param_data(bound_params, label->paramid);
+#else
+    ParamExternData *param_data = get_param_data(bound_params, label->paramid - 1);
```

Comment:
Better to conditionally define a static `PARAM_OFFSET` constant and then use it across the board (I think this snippet will be required in more places).

Reply:
Hm, this could mislead that the offset can be arbitrarily large, while it is an off-by-one situation only.

I'm trying to solve this a little differently, which I'll show in the `release/elm` PR, which will attempt to put this `paramid` logic in one place, so there shouldn't be a need to duplicate it around. Let's continue there.